### ISSUE TITLE
RAM coefficient value - implementation vs. HotCarbon paper

### ISSUE THREAD

**Issue by davidkopp** [Boavizta/boaviztapi#298]
At the moment, the coefficient value 0.284 W/GB is used in the implementation.

Source code: https://github.com/Boavizta/boaviztapi/blob/4af9f116170be39e919b1d95dc4bd02b65afd93b/boaviztapi/model/consumption_profile/consumption_profile.py#L30
Docs: https://doc.api.boavizta.org/Explanations/components/ram/#determining-the-parameters

In the paper that was recently published during the HotCarbon conference a different value is mentioned:
> For a given model, the TechPowerUp SSD specs database [35] can  be used to estimate the idle and active (averaged) power consumption values. For RAM banks, we use the averaged values from [33]:  0.19 W /GB and 0.54 W /GB in idle and active states, respectively. By default, RAM banks are constantly considered in an active state.

Source: Simon, T. et al. (2024) ‘BoaviztAPI: a bottom-up model to assess the environmental impacts of cloud services’, in. HotCarbon 2024. Available at: https://hotcarbon.org/assets/2024/pdf/hotcarbon24-final74.pdf

This seems to me as an inconsistency. Which value is more accurate?

**Comment by da-ekchajzer** [comment]
@tibosmn @samuelrince do you remember ? Should we update the value ?

**Comment by tibosmn** [comment]
If I recall correctly, we decided to use a value sourced from the literature rather than one derived from the modeling process, which primarily show a static consumption anyway.
I'd update the value but let the final call to @samuelrince ;)

**Comment by samuelrince** [comment]
Indeed, the current value was an average computed from @github-benjamin-davy AWS dataset. But, we chose a different source in the paper. I think we should update the value in the API.

To do:

- [ ] Change RAM consumption profile to only support idle and active states
- [ ] Allow user to change, idle and active power consumption in addition to a utilization ratio
- [ ] Compute that ratio for archetypes depending on the instance type
- [ ] Have a default value set to 50%, maybe?
